pkg/server/container_create.go

```diff
@@ -35,6 +36,14 @@ import (
 	"github.com/kubernetes-incubator/cri-containerd/pkg/util"
 )
 
+// ProfileNamePrefix is the prefix for loading profiles on a localhost. Eg. AppArmor localhost/profileName.
+const (
```

Comment:
We should get these constants from https://github.com/kubernetes/kubernetes/issues/51747.
Add TODO for it.

Reply:
the todo is 4 lines down...

Reply:
I mean we should get the first 2 constants from kubernetes package, which we haven't done yet.